code/index.html

```diff
@@ -4,9 +4,18 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Project 4: Pizzabot</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Pizzabot 🍕</title>
   </head>
   <body>
-    <h1>This is your Pizza Bot</h1>
+ 
+  <section id="feedback">
+    <h1>Pizzeria Da Matteo e Pierina</h1>
+    <button onclick="buttonClicked()"> ORDER </button>
```

Comment:
It would be good it the function describe what will happen. `buttonClicked()` doesn't say so much of what will happen when you click on this button (which we know will happen because the element is a button that has `onclick` on it)